modules/ROOT/pages/basic-alerts.adoc

```diff
@@ -40,4 +40,79 @@ The following levels of severity are available for basic alerts:
 
 include::partial$include-alert-severity.adoc[leveloffset=+1]
 
-// add basic alert email templates
+[[alert-variables]]
+== Basic Alert Email Template
+
+You can use the following variables in a basic alert email's *Subject* and *Message* fields.
+For the description of each variable, see <<var-defs>>.
+
+[%header,cols="25a,50a,20a"]
+.Variables for Alerts
+|===
+| Alert Type | Alert | Applicable Variables
+2+| All Alerts | `${resource}` `${severity}` `${state}`
+.7+| CloudHub App Alerts |*CPU usage - CloudHub* .2+|`${operator}` `${value}`
+| *Memory usage - CloudHub*
+| *Custom application notification* | `${priority}` `${message}`
+| *Deployment failed* |  `${user}` `${message}`
+| *Deployment success* .2+| `${user}`
+| *Worker not responding*
+| *Exceeds event traffic threshold* | `${period}`
+
+
+.4+|Hybrid App Alerts | All Hybrid App Alerts| `${name}`
+| *Number of Mule messages* .3+| `${value}` `${threshold}`
+| *Number of errors*
+| *Response time*
+
+.5+|Server Alerts | All Server Alerts | `${name}`
+| *Server CPU usage* .4+|  `${value}` `${comparison}` `${threshold}`
+| *Server Load average*
+| *Server Memory average*
+| *Server Thread count*
+
+.9+| Server Group Alerts| All Server Group Alerts | `${name}`
```

Comment:
For hybrid servers, server groups and clusters, these are the metrics we can create alerts on:

Cpu Utilization
Memory Utilization
Thread Count